src/main/java/algorithms/curated170/medium/minimumknightmoves/MinimumKnightMovesBFS.java

```diff
@@ -0,0 +1,56 @@
+package algorithms.curated170.medium.minimumknightmoves;
+
+import java.util.ArrayDeque;
+import java.util.Deque;
+
+public class MinimumKnightMovesBFS {
```

Comment:
BFS solution reminded me of the biosonar that bats use to measure distance:
https://www.youtube.com/watch?v=laeE4icRYp4